helm/raster/templates/nginx-deployment.yaml

```diff
@@ -79,10 +67,16 @@ spec:
           image: {{ $cloudProviderDockerRegistryUrl }}{{ .repository }}:{{ .tag }}
           {{- end }}
           imagePullPolicy: {{ .Values.imagePullPolicy }}
+          env:
+          - name: DOMAIN
```

Comment:
Could you let me know why you added the domain? in nginx.conf file, you already set the domain variable directly from the values.yaml and not from env.